src/garage/experiment/local_tf_runner.py

```diff
@@ -311,11 +319,10 @@ def train(self,
             The average return in last epoch cycle.
 
         """
-        assert self.has_setup, ('Use Runner.setup() to setup runner before '
-                                'training.')
+        assert self.has_setup, 'Use setup() to setup runner before training.'
```

Comment:
if you are detecting a user error, rather than asserting an invariant, raise and exception.